Minor fix for render issues

Dayitva opened the pull request:
## Contributing

If you want to add a project to this list, please make sure that:

- [x] You read the [contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2020/blob/master/CONTRIBUTING.md)
- [x] The repository has hacktoberfest as topic and counts for hacktoberfest event
- [x] The project can't be just an "add your name" or simply + 1 contribution to hacktoberfest
- [x] Place them in alphabetical order
- [x] One repository per PR/contribution - _avoid conflicts and simplify review_
- [x] Your pull request is not spammy just adding an empty line, otherwise it will be marked as invalid and not count to hacktoberfest.

If they meet the requirements above, feel free to create a PR. Happy Hacking! :sparkles:

OtacilioN reviewed (approved):
Awesome, thanks for contributing!

OtacilioN commented:
Also we have some conflicts here, can you please fix it?

Dayitva commented:
> Also we have some conflicts here, can you please fix it?

Fixed the conflict. An earlier PR which was merged had already fixed one of the issues I corrected in my PR.
